RestAPI.cs

```diff
@@ -175,7 +175,8 @@ public static string SerializeJSon<T>(T t)
         {
             DataContractJsonSerializerSettings settings = new DataContractJsonSerializerSettings()
             {
-                DateTimeFormat = new DateTimeFormat("yyyy-MM-ddTHH:mm:ssZ")
+                DateTimeFormat = new DateTimeFormat("yyyy-MM-ddTHH:mm:ssZ"),
+                UseSimpleDictionaryFormat = true

```

Comment:
I assume that we need to set UseSimpleDictionaryFormat to true in DeserializeJSon function as well?